xbmc/video/VideoInfoScanner.cpp

```diff
@@ -744,7 +745,15 @@ namespace VIDEO
       if (loader)
       {
         pItem->GetVideoInfoTag()->Reset();
-        result = loader->Load(*pItem->GetVideoInfoTag(), false);
+        CVideoInfoTag& infoTag = *pItem->GetVideoInfoTag();
+        result = loader->Load(infoTag, false);
+
+        // only import some properties if advancedsettings.xml says so
```

Comment:
nitpick, maybe "keep some properties only if advancedsettings.xml says so" would be clearer

Reply:
Thanks, updated :+1: